.github/workflows/build.yaml

```
@@ -18,6 +18,10 @@ jobs:
         uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v2
         with:
           go-version: '1.22.5'
+      - name: Install Wire
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:24: Double quote to prevent globbing and word splitting

(shellcheck)